### ISSUE TITLE
`valid-title`: Should allow variables

### ISSUE THREAD

**Issue by fisker** [jest-community/eslint-plugin-jest#669]
```js
test(someTitle, () => {

})
```

Should not reported.

**Comment by G-Rath** [comment]
This is supported by enabling the [`ignoreTypeOfDescribeName`](https://github.com/jest-community/eslint-plugin-jest/blob/master/docs/rules/valid-title.md#ignoretypeofdescribename) option.

**Comment by fisker** [comment]
Thanks, I'll try.

**Comment by fisker** [comment]
Sorry, I tried `ignoreTypeOfDescribeName: true`, it can't solve problem, seems it only work for `describe`

https://github.com/jest-community/eslint-plugin-jest/blob/57ef99f1fabf4a03c893d92e94a494b1289fa362/src/rules/valid-title.ts#L184

**Comment by G-Rath** [comment]
Oh yeah sorry quite right - the actual solution is to disable the rule.

The point of `valid-title` is to valid string titles, so there's no point in using it if you're not going to have string titles.
You can use the standard `eslint-disable` comments to control applying the rule on a case-by-case :)

**Comment by fisker** [comment]
Okay, thank you for clearify this.
